src/main/java/org/gridsuite/study/server/service/PccMinService.java

```diff
@@ -37,17 +39,18 @@
 import org.springframework.data.domain.Pageable;
 
 import static org.gridsuite.study.server.StudyConstants.*;
-import static org.gridsuite.study.server.StudyException.Type.PCC_MIN_NOT_FOUND;
-import static org.gridsuite.study.server.StudyException.Type.PCC_MIN_RUNNING;
+import static org.gridsuite.study.server.StudyException.Type.*;
+import static org.gridsuite.study.server.StudyException.Type.CREATE_PCC_MIN_PARAMETERS_FAILED;
+import static org.gridsuite.study.server.utils.StudyUtils.handleHttpError;
 
 /**
  * @author Maissa SOUISSI <maissa.souissi at rte-france.com>
  */
 @Service
 public class PccMinService extends AbstractComputationService {
     static final String RESULT_UUID = "resultUuid";
-    static final String FILTER_UUID = "filterUuid";
     static final String BUS_ID = "busId";
+    private static final String PARAMETERS_URI = "/parameters/{parametersUuid}";
```

Comment:
```suggestion
    private static final String PCC_MIN_URI = DELIMITER + PCC_MIN_API_VERSION;
    private static final String PARAMETERS_URI = PCC_MIN_URI + "/parameters";
```
then use those constants everywhere ?